content: Add new land/planet images

petervdmeer opened the pull request:
**Content (Artwork / Photographs)**

## Summary
Some additional spaceport/planet photos for use in the game; pictures taken by me. I didn't have any specific planets in mind for the photos, but #6446 suggests that they might be useful somewhere.
Some pictures might also be useful for alien worlds after some photo-editting, especially the pictures with silhouettes of mills or towers; those could be altered to spaceports or alien structures.

## Save File
N/A

## PR Checklist
 - [x] I updated the copyright attributions
 - [x] I uploaded the necessary image, blend, and texture assets here: (N/A, photographs are in this PR)
 - [ ] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: TODO (I have the photographs generated, just didn't upload them yet)

Hurleveur reviewed (commented):
you got some skills! some of them may be too close for spaceport images but overall it's a great improvment!
that said this should probably be modified to use the images at the proper places, there is an issue by amazinite tracking which spaceport have no images

or there could be a change to make it so you can have multiple landing images per planet?

Zitchas commented:
Nice photography! I think those should do wonderfully to add to the variety of images we have and eliminate a bunch of duplicate usage.

That being said, I'm not sure about the one with the cars visible in it. Forest7, I think? If I recall correctly, we have declined images that showed actual cars in them. (I guess preserving the idea that *maybe* we have stuff that is better by 31st century?)

petervdmeer commented:
Thanks! I removed forest7.jpg, because regular cars indeed don't really fit well when you already have repulsors.

@Hurleveur Which of the photos do you feel are too close for spaceport images?

Hurleveur commented:
> Thanks! I removed forest7.jpg, because regular cars indeed don't really fit well when you already have repulsors.
> 
> @Hurleveur Which of the photos do you feel are too close for spaceport images?

water19 sea19 hills10 forest8 forest6 fields20 but you don't have to worry, I'm not that experienced in this game, also we can change it to have some images be closer than usual, no need for always having a landscape far away

Pointedstick commented:
These look great to me except for `images/land/city19-riverside.jpg` and `images/land/city20-riverside.jpg` which I think might be too recognizable as real-world cities. This is kind of already a problem with a bunch of city images, but I wouldn't want to worsen it, and I think images of large, easily recognizable landmarks would do so.

Zitchas commented:
Captain: "this looks familiar... Did someone intentionally try to recreate their home town when they colonized this world?"

Jokes aside, it is probably a question of "Is the benefit of a nice picture outweighed by the fact some people might recognize it?" Probably not, in the case of really distinctive landmarks, but if it is just locals that will recognize it, I don't have a problem.

Amazinite reviewed (commented):
Landscape images typically avoid having a foreground right in front of the camera, instead being a picture from far away, but when there is a foreground it's uniform and devoid of any objects. I'd say an example of an image with a uniform foreground from this PR would be fields14 and 15. But some of these pictures have a foreground that isn't uniform that I'd say don't really fit as landscape images. I'd say we should also avoid having roads in the landscape images.

Images that I'd remove:
* city20 (foreground)
* fields16 (foreground)
* fields17 (road/path)
* fields19 (road/path)
* fields20 (road/path)
* fields21 (foreground)
* fields23 (road/path)
* fields30 (road/path)
* forest6 (too up close)
* forest8 (road/path, also you skipped 7)
* valey16 (not a fan of the glare from the sun)

fields18 is on thin ice with those plants in the bottom left but it's a great image otherwise so I'll let it slide.

A lot of these have human architecture in them. That really limits where we can use them, so some of these images might go unused simply because we have no where fitting to put them.

petervdmeer commented:
Image `forest7` wasn't skipped, but I deleted it quite late in the upload process because I noticed a shadow from the person taking the photo in the picture.

petervdmeer commented:
> These look great to me except for `images/land/city19-riverside.jpg` and `images/land/city20-riverside.jpg` which I think might be too recognizable as real-world cities. This is kind of already a problem with a bunch of city images, but I wouldn't want to worsen it, and I think images of large, easily recognizable landmarks would do so.

The bridge on city19 was indeed quite unique and recognizable.

petervdmeer commented:
> Image `forest7` wasn't skipped, but I deleted it quite late in the upload process because I noticed a shadow from the person taking the photo in the picture.

Sorry, wrong picture. Forest 7 was removed based on a review remark above (and I removed other pictures because of shadows).

Amazinite reviewed (changes requested):
Sea20 and sea21 seem too similar to me, so I'd say pick one to keep and toss the other one; between the two I prefer sea21.
Something about forest11, hills10, hills11, water19, and water20 tells me to toss them.

Otherwise, what remains look good to go.
